package.json

```diff
@@ -28,13 +28,28 @@
     "Red Hat"
   ],
   "extensionDependencies": [
-	"asciidoctor.asciidoctor-vscode"
+    "asciidoctor.asciidoctor-vscode"
   ],
   "activationEvents": [
     "onLanguage:asciidoc"
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "configuration": {
+      "title": "redhat.asciidoc",
+      "properties": {
+        "redhat.asciidoc.remoteAttributeDefinitions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of files from which attributes will be retrieved. It is up to users to ensure that they will be available. By default, configured with some Red Hat specific attribute defintion files.",
+          "default": [
+            "https://gitlab.cee.redhat.com/red-hat-jboss-web-server-documentation/jws-documentation/-/raw/master/docs/topics/templates/document-attributes.adoc?inline=false"
```

Comment:
for now, we could add all the different files. In description it is listed from which file the attribute is coming from. When there will be a single one, it will be a single one :-)